core/src/avm2/globals/flash/media/MicrophoneEnhancedOptions.as

```diff
@@ -0,0 +1,22 @@
+// The initial version of this file was autogenerated from the official AS3 reference at
+// https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/media/MicrophoneEnhancedOptions.html
+// by https://github.com/golfinq/ActionScript_Event_Builder
+// It won't be regenerated in the future, so feel free to edit and/or fix
+
+package flash.media
+{
+    public final class MicrophoneEnhancedOptions
+    {
+        // Specifies the echo path (in milliseconds) used for acoustic echo cancellation.
+        public var echoPath: int;
+
+        // Indicates whether the Microphone input detected a voice.
+        public var isVoiceDetected: int;
+
+        // Controls enhanced microphone mode.
+        public var mode: String;
```

Comment:
From the Flash docs:
Controls enhanced microphone mode. The default value is `MicrophoneEnhancedMode.FULL_DUPLEX` for all microphones that aren't USB. The default value for USB microphones is `MicrophoneEnhancedMode.HALF_DUPLEX`. See MicrophoneEnhancedMode for possible values and descriptions.

Reply:
I've set this to the default option and added a stub. Please check if I'm doing this one correct.